core/config.py

```diff
@@ -178,4 +178,15 @@
 
     # Will be set to viur.core.version.__version__ in viur.core.__init__
     "viur.version": None,
+
+    # Admin Parts
+    "admin.name": "Viur Administartion--Testr",
+    "admin.logo": "",
+    "admin.login.background": "/vi/login-background-2.jpg",
```

Comment:
Please provide a more reasonable default configuration here:
```suggestion
    # Administration tool configuration
    "admin.name": "ViUR",  # I would suggest to use utils.projectID here...
    "admin.logo": "",
    "admin.login.background": "",
```